internal/server/property_label.go

```diff
@@ -52,6 +52,10 @@ func (s *Server) GetPropertyLabels(ctx context.Context,
 	}
 	result := map[string]*PropLabelCache{}
 	for dcid, data := range dataMap {
+		if data == nil {
+			result[dcid] = nil
+			continue
+		}
```

Comment:
So, other callsites of bigTableReadRowsParallel don't have this problem?

Reply:
There are a bunch of type conversion unchecked. Fixed them all in https://github.com/datacommonsorg/mixer/pull/440